ext/intl/normalizer/normalizer_class.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | https://www.php.net/license/3_01.txt                                 |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Authors: Ed Batutis <ed@batutis.com>                                 |
+   +----------------------------------------------------------------------+
+ */
+
+#ifdef __cplusplus
```

Comment:
Well this is a .cpp file, so surely this is true? (same below)

Reply:
ah yes I probably copied/pasted from somewhere else :)